spec/features/new_peep_spec.rb

```diff
@@ -0,0 +1,25 @@
+feature 'Create Peep:' do
+  scenario 'A user can create a new Peep message' do
+    create_account('Test', 'mrtest', 'test@test.uk', 'Testing')
+    write_new_peep('hello world')
+    expect(Peep.all.length).to eq(1)
+    expect(page.current_url.ends_in?('/peeps')).to be(true)
+    expect(page).to have_content('hello world')
+    expect(page).to have_content('mrtest')
```

Comment:
Feels like you are testing multiple things in the same test here. A test should usually only have one reason to fail.